src/main/java/com/google/sps/servlets/utils/Range.java

```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.sps.servlets;
+package com.google.sps.servlets.utils;
 
-/** Range for sorting sentiment scores */
+/** Range for the start and end point of sentiment scores. */
 public class Range {
   private double inclusiveStart;
```

Comment:
I am not exactly sure what happens when you convert BigDecimal to double here again.
The best way and cleanest way to do this is probably use int or enums with defined ranges.
It is a little late for this CL, so nothing needs to be changed here.

Reply:
resolved in discussion and add TODO